front/src/Phaser/Entity/PlayableCaracter.ts

```diff
@@ -1,13 +1,20 @@
 import {getPlayerAnimations, playAnimation, PlayerAnimationNames} from "../Player/Animation";
 import {ActiveEventList, UserInputEvent} from "../UserInput/UserInputManager";
 import {SpeechBubble} from "./SpeechBubble";
+import BitmapText = Phaser.GameObjects.BitmapText;
 
 export class PlayableCaracter extends Phaser.Physics.Arcade.Sprite {
     private bubble: SpeechBubble;
+    private playerName: BitmapText;
 
-    constructor(scene: Phaser.Scene, x: number, y: number, texture: string, frame?: string | number) {
+    constructor(scene: Phaser.Scene, x: number, y: number, texture: string, name: string, frame?: string | number) {
         super(scene, x, y, texture, frame);
 
+        // Yes, I know, I'm declaring a sprite inside a sprite. ARP, save me from this madness :)
```

Comment:
I don't know if it is that bad. I did basically the same thing for the  player bubble or the textInput underline. The player name render is the responsibility of the PlayableCarater class and it have to be a GameObject so it kinda of make sense to instantiate it here. In the end, it is still an object of the scene.

Reply:
After reflexion, this can probably be solved by using containers. I ll look into it